docs/content/quickstart/quick-start.md

```diff
@@ -81,5 +81,5 @@ You can learn more about configuring your server and loading environment variabl
 
 - Install our [Next.js](../starters/nextjs-medusa-starter) or [Gatsby](../starters/gatsby-medusa-starter) storefronts to set up your ecommerce storefront quickly.
 - Install the [Medusa Admin](../admin/quickstart.md) to supercharge your ecommerce experience with easy access to configurations and features.
-- Check out the [API reference](https://docs.medusajs.com/api) to learn more about available endpoints available on your Medusa server.
+- Check out the [API reference](https://docs.medusajs.com/api/admin) to learn more about available endpoints available on your Medusa server.
```

Comment:
```suggestion
- Check out the [API reference](https://docs.medusajs.com/api/store) to learn more about available endpoints available on your Medusa server.
```

Reply:
I was thinking if admin is the right place for this 😄